docs/handbook/image-file-formats.rst

```diff
@@ -521,6 +521,19 @@ The :py:meth:`~PIL.Image.Image.save` method supports the following options:
 
     .. versionadded:: 2.5.0
 
+**streamtype**
+    Allows storing images without quantization and Huffman tables, or with
+    these tables but without image data.  This is useful for container formats
+    or network protocols that handle tables separately and share them between
+    images.
+
+    * ``0`` (default): interchange datastream, with tables and image data
+    * ``1``: abbreviated table specification (tables-only) datastream
+
+      .. versionadded:: 10.2.0
```

Comment:
How it looks like on the page? From the source it's not clear to me that streamtype==1 was added. Probably would be better just to add it in the text, rather than use special formatting.

Reply:
https://pillow--7601.org.readthedocs.build/en/7601/handbook/image-file-formats.html

<img width="1009" alt="preview" src="https://github.com/python-pillow/Pillow/assets/3112309/5ce69656-bd18-43e0-9a64-cd0df0a33f0b">